google/cloud/pubsub/internal/tracing_helpers.h

```diff
@@ -29,6 +32,21 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 // Create start span options for a root span.
 opentelemetry::trace::StartSpanOptions RootStartSpanOptions();
 
+/// Create a list of links with @p span context if compiled with open telemetery
+/// ABI 2.0.
+std::vector<
+    std::pair<opentelemetry::trace::SpanContext,
+              std::vector<std::pair<opentelemetry::nostd::string_view,
+                                    opentelemetry::common::AttributeValue>>>>
```

Comment:
```suggestion
using TracingAttributes =  std::vector<std::pair<
    opentelemetry::nostd::string_view, opentelemetry::common::AttributeValue>>;

/// Create a list of links with @p span context if compiled with open telemetery
/// ABI 2.0.
std::vector<std::pair<opentelemetry::trace::SpanContext, TracingAttributes>>
```

Reply:
I didn't know if we wanted those aliases in the pubsub_internal namespace. I don't think it should matter too much though since it is internal. Done.